docs/user/DockerShell.md

```diff
@@ -1,4 +1,63 @@
-```{include} ../../flow/util/README.md
+Building example design using Docker image
+==========================================
+
+The `docker_shell` script is used as a prefix to launch a command using a OpenROAD docker image.
+
+Also, the current working directory is mapped into the Docker image using the current user's credentials.
+
+Build docker image
+------------------
+
+First build the docker image:
+
+```
+cd OpenROAD-flow-scripts
+sudo ./setup.sh
```

Comment:
```suggestion
```

Reply:
For docker builds, the only requirement should be docker, which is not installed by `setup.sh`. Right?

Reply:
Yes, you are correct. Should we add a disclaimer asking users to install Docker first?

Reply:
For building, we should point to `BuildWithDocker.md` as the single source for building with docker. In the `DockerShell.md`, there should be no/minimal mention of building; it should focus on running the flow using the helper script.

Reply:
Should we remove the first section on "First, build the docker image"?